src/frontend/components/login/Login.vue

```diff
@@ -30,35 +31,56 @@
                 name="password"
                 :error="!!errors.password"
                 :error-messages="errors.password"
-                prepend-icon="lock"
+                prepend-icon="fingerprint"
                 ref="password"
                 onfocus="this.select()"
                 type="password"></v-text-field>
             </v-card-text>
 
             <v-card-actions>
-              <nuxt-link to="/passwordReset">Forgot password?</nuxt-link>
+              <v-btn type="submit" style="color: green;" v-on:click="guestHome"  
```

Comment:
We should try to avoid inline `style` tags. Instead vuetify has a good way to do this same thing with classnames that they define for us.

```suggestion
              <v-btn type="submit" class="green--text" v-on:click="guestHome"  
```